pkgs/servers/http/nginx/modules.nix

```diff
@@ -488,10 +488,10 @@ in
       name = "video-thumbextractor";
       owner = "wandenberg";
       repo = "nginx-video-thumbextractor-module";
-      rev = "0.9.0";
-      sha256 = "1b0v471mzbcys73pzr7gpvzzhff0cva0l5ff32cv7z1v9c0ypji7";
+      rev = "92b80642538eec4cfc98114dec5917b8d820e912";
```

Comment:
Why did you change that?

Reply:
I forgot to report what I wrote on the issue about ffmpeg 3: https://github.com/NixOS/nixpkgs/issues/120705#issuecomment-845820604